test/test_polynomial.cpp

```diff
@@ -298,3 +298,26 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(test_odd_even, T, all_test_types)
     BOOST_CHECK_EQUAL(odd(b), false);
     BOOST_CHECK_EQUAL(even(b), true);
 }
+
+BOOST_AUTO_TEST_CASE_TEMPLATE( test_pow, T, all_test_types )
+{
+    polynomial<T> a(d3a.begin(), d3a.end());
+    polynomial<T> const one = identity_element(std::multiplies< polynomial<T> >());
```

Comment:
+1, I'd prefer to simplify that, though it doesn't make much difference in a test case I admit.